camera_node:
  camera_type: "CSI"
  fps: "30"
  resolution_H: 1080
  resolution_W: 1920


mic_audio_node:
  # port_name: "/dev/ttyUSB0"
  baud_rate: 115200
  block_size: 1024
  sample_rate: 16000
  channels: 1

stt_node:
  model_path: "../../models/vosk_model"
  sample_rate: 16000
  buffer_size: 1024  
  queue_size: 50
  audio_topic: "mic_audio"
  text_topic: "stt_text"
  silence_threshold: 0.1
  silence_duration: "0.5"

agent_node:
  speaker: ["speaker1", "speaker2"]
  speaker_history: ["history1.json", "history2.json"]


executor_node:
  tasks : [
    {"task_name": ["msg_type","topic_name"]},
    {"task2_name": ["msg2_type","topic_name2"]},
    {"task3_name": ["msg3_type","topic_name3"]},
  ]


sllidar_node:
  channel_type: "serial"
  port_name: "/dev/ttyUSB0"
  serial_baudrate: 460800
  frame_id: "laser"
  inverted: false
  angle_compensate: true
  scan_mode: "standard"

common: 
  K: [
    fx, 0.0, ux,
    0.0, fy, vy,
    0.0, 0.0, 0.0
  ]
  
  D: [
    k1, k2, p1, p2, k3
  ]
